### PR TITLE
Feature/implement work time record

### PR DESCRIPTION
**Pull request by salman7C9**
- added a WorkTimeRecord class to keep track of the start and end time of each record
- added test for duration
- updated main
- removed obsolete code

### CODE REVIEW

File: WorkTracker/Tests/WorkTimeRecordTest.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```
Comment (Noxsense) [inline WorkTracker/Tests/WorkTimeRecordTest.cs]: Unused imports?

File: WorkTracker/Build/WorkTimeRecord.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Dynamic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```
Comment (Noxsense) [inline WorkTracker/Build/WorkTimeRecord.cs]: Unused Imports?

**Review by Noxsense — APPROVED** [review]
Looks fine, please clean out the unused imports.

File: WorkTracker/Tests/WorkTimeRecordTest.cs (hunk 1 above)
Comment (salman7C9) [reply WorkTracker/Tests/WorkTimeRecordTest.cs]: removed unused imports

**Review by Noxsense — APPROVED** [review]

**Pull request merged**